### ISSUE TITLE
Hierarchical search and browse

### ISSUE THREAD

**Issue by wsalesky** [srophe/srophe-eXist-app#971]
See related issues:
https://github.com/srophe/srophe-app-data/issues/463
https://github.com/srophe/srophe-app-data/issues/710

**Comment by davidamichelson** [comment]
Search looks good.

For browse and search, in the summary view for "composite" URI records, include a sentence with a count like this:

This literary tradition comprises [count] texts with separate URIs.

**Comment by davidamichelson** [comment]
For places here is a draft:

This composite place includes [count] more narrowly conceived places.

**Comment by wsalesky** [comment]
Will there be/can there be more then one value in  `@passive` or is there always only one 'parent'?

`<relation active="http://syriaca.org/work/410" ref="skos:broadMatch" passive="http://syriaca.org/work/2057"/>
`

**Comment by nathangibson** [comment]
Hmm, @davidamichelson and I haven't specifically discussed that. Usually there would only be one, but I can conceive of a work that might belong to multiple literary traditions, such as a saint's life that blends material from multiple other lives. 

Is this a we-can-cross-that-bridge-when-we-come-to-it sort of situation, or is it best to future-proof it now? 
And does it make a difference in query speed if additional parents were each in their own relation elements rather than as multiple passive values?

**Comment by wsalesky** [comment]
It would be fine if they each had their own element, it is the text parsing that eats up processing time. Again, rdf may also solve this for us.

**Comment by nathangibson** [comment]
@davidamichelson What would you think about us requiring that if a URI belongs to multiple composite URIs, that we put in multiple `<relation>` elements (instead of simply multiple `@passive` values)? We might even be able to write such a rule into the ODD, though it'd be a little complicated.

**Comment by dlschwartz** [comment]
@nathangibson You can easily allow only one value for the passive attribute and the encoder would then just have to know that she is supposed to add another relation element.

**Comment by nathangibson** [comment]
Good to know, thanks! Can we restrict to only relations that have a ref=“skos:broadMatch”?

**Comment by dlschwartz** [comment]
Yes, but you would have to use schematron in your ODD. You would apply a text that would only allow one value for the passive attribute if there is also a ref element with a value of skos:broadMatch. Multiple values on the passive attribute would render an error in Oxygen. I can help or you can check out Syd Bauman's slides, etc. for the course I took at DHSI last summer (toward the bottom): [http://www.wwp.northeastern.edu/outreach/resources.html]

**Comment by wsalesky** [comment]
Let's hold off on that until after I test RDF, which may end up. Ring faster.

**Comment by davidamichelson** [comment]
Here is a sentence for places:

This composite place URI combines [count] specific places.

**Comment by davidamichelson** [comment]
I think we will want different handling for this in Places and Works. For places we will want all the children to show in browse list and also independently but this will be fine because there are not many of the.

**Comment by wsalesky** [comment]
This is causing some issues and will need to be reworked. Expect to see some odd search results over the next week as I sort it out.
